lib/src/components/material_ripple/material_ripple.dart

```diff
@@ -151,12 +151,15 @@ void _applyFallbackAnimation(
 ///
 /// By default, the ripple is centered on the point of contact.
 @Component(
-    selector: 'material-ripple',
-    template: '',
-    encapsulation: ViewEncapsulation.None,
-    styleUrls: const ['material_ripple.scss.css'],
-    changeDetection: ChangeDetectionStrategy.Detached,
-    visibility: Visibility.none)
+  selector: 'material-ripple',
+  template: '',
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: const ['material_ripple.scss.css'],
+  changeDetection: ChangeDetectionStrategy.Detached,
+  visibility: Visibility.none,
```

Comment:
What does setting the visibility to `none` do? Don't we want components to be visible by default?

Reply:
Looks like it makes it so that this component is not available for dependency injection.
https://webdev.dartlang.org/api/angular/angular/Directive/visibility 

@ferhatb made this change and would know more.